edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/RegistryCredentials.cs

```diff
@@ -24,6 +32,9 @@ public RegistryCredentials(string address, string username, string password)
         [JsonProperty(Required = Required.Always, PropertyName = "password")]
         public string Password { get; }
 
+        [JsonProperty(PropertyName = "rootKey")]
```

Comment:
Is it required like address and password are?

Reply:
No. it is not required.